projects/packages/jetpack-mu-wpcom/src/features/custom-css/custom-css.php

```diff
@@ -987,8 +987,7 @@ public static function update_custom_css_data( $args ) {
 		public static function editor_max_image_size( $dims, $size = 'medium', $context = null ) {
 			list( $width, $height ) = $dims;
 
-			// @phan-suppress-next-line PhanUndeclaredClassInCallable
-			if ( class_exists( 'Jetpack' ) && is_callable( 'Jetpack::get_content_width' ) && 'large' === $size && 'edit' === $context ) {
+			if ( class_exists( 'Jetpack' ) && 'large' === $size && 'edit' === $context ) {
```

Comment:
Looks like a reasonable change, but doesn't seem to have anything to do with the PHP 7.2 change. Same for the similar stuff below.

I wonder if we even need the `class_exists( 'Jetpack' )` test? This seems to have been added in dd3b1d83735 to try fixing static analysis.

Reply:
Changed PR title.

As for whether we need it, I wasn't confident enough in the load order and knowledge of the environment to know if there's an edge case where the class might not exist. 😓